packages/message-box/message-box.scss

```diff
@@ -8,6 +8,10 @@
     padding: $component-spacing--xl;
     background-color: $snøhvit;
 
+    &--full {
```

Comment:
Maybe `&--full-width` to make it even clearer?

Reply:
agreed